tdp/cli/commands/plan/test_dag.py

```diff
@@ -2,23 +2,21 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-from pathlib import Path
-
 from click.testing import CliRunner
 
-from tdp.cli.commands.init import init
+from tdp.cli.commands.conftest import tdp_init_args
 from tdp.cli.commands.plan.dag import dag
 
 
-def test_tdp_plan_dag(collection_path: Path, db_dsn: str, vars: Path):
+def test_tdp_plan_dag(
+    tdp_init: tdp_init_args,
+):
     base_args = [
         "--collection-path",
-        collection_path,
+        tdp_init.collection_path,
         "--database-dsn",
-        db_dsn,
+        tdp_init.db_dsn,
     ]
```

Comment:
No need for those intermediary arrays as only one command is executed from now.

Reply:
done